sdk/storage/storage-queue/src/QueueServiceClient.ts

```diff
@@ -540,4 +441,133 @@ export class QueueServiceClient extends StorageClient {
+        message: e.message
+      });
+      throw e;
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Sets properties for a storage account’s Queue service endpoint, including properties
+   * for Storage Analytics, CORS (Cross-Origin Resource Sharing) rules and soft delete settings.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/set-queue-service-properties
+   *
+   * @param {Models.QueueServiceProperties} properties
+   * @param {ServiceGetPropertiesOptions} [options] Options to set properties operation.
+   * @returns {Promise<Models.ServiceSetPropertiesResponse>} Response data for the Set Properties operation.
+   * @memberof QueueServiceClient
+   */
+  public async setProperties(
+    properties: Models.QueueServiceProperties,
+    options: ServiceGetPropertiesOptions = {}
+  ): Promise<Models.ServiceSetPropertiesResponse> {
+    const { span, spanOptions } = createSpan(
+      "QueueServiceClient-setProperties",
+      options.spanOptions
+    );
+    try {
+      return this.serviceContext.setProperties(properties, {
+        abortSignal: options.abortSignal,
+        spanOptions
+      });
+    } catch (e) {
+      span.setStatus({
+        code: CanonicalCode.UNKNOWN,
+        message: e.message
+      });
+      throw e;
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Retrieves statistics related to replication for the Queue service. It is only
+   * available on the secondary location endpoint when read-access geo-redundant
+   * replication is enabled for the storage account.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/get-queue-service-stats
+   *
+   * @param {ServiceGetStatisticsOptions} [options] Options to get statistics operation.
+   * @returns {Promise<Models.ServiceGetStatisticsResponse>} Response data for get statistics the operation.
+   * @memberof QueueServiceClient
+   */
+  public async getStatistics(
+    options: ServiceGetStatisticsOptions = {}
+  ): Promise<Models.ServiceGetStatisticsResponse> {
+    const { span, spanOptions } = createSpan(
+      "QueueServiceClient-getStatistics",
+      options.spanOptions
+    );
+    try {
+      return this.serviceContext.getStatistics({
+        abortSignal: options.abortSignal,
+        spanOptions
+      });
+    } catch (e) {
+      span.setStatus({
+        code: CanonicalCode.UNKNOWN,
+        message: e.message
+      });
+      throw e;
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Creates a new queue under the specified account.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/create-queue4
+   *
+   * @param {QueueCreateOptions} [options] Options to Queue create operation.
+   * @returns {Promise<Models.QueueCreateResponse>} Response data for the Queue create operation.
+   * @memberof QueueServiceClient
+   */
+  public async createQueue(
+    queueName: string,
+    options: QueueCreateOptions = {}
+  ): Promise<Models.QueueCreateResponse> {
```

Comment:
Newly added.

Reply:
Shouldn't this have a tracing span for it?

Reply:
Logged and issue #5595